ci/jobs/build_clickhouse.py

```diff
@@ -181,7 +183,13 @@ def main():
         )
         run_shell("sccache stats", "sccache --show-stats")
         if build_type in (BuildTypes.AMD_TIDY, BuildTypes.ARM_TIDY):
-            run_shell("clang-tidy-cache stats", "clang-tidy-cache --show-stats")
+            run_shell("clang-tidy-cache stats", "clang-tidy-cache.py --show-stats")
+            clang_tidy_cache_log = "./ci/tmp/clang-tidy-cache.log"
+            Shell.check(f"cp /tmp/clang-tidy-cache.log {clang_tidy_cache_log}")
+            files.append(clang_tidy_cache_log)
+            run_shell("clang-tidy-cache.log stats", f'echo "$(grep "exists in cache" {clang_tidy_cache_log} | wc -l) in cache\n'
+                                                    f'$(grep "does not exist in cache" {clang_tidy_cache_log} | wc -l) not in cache"')
+            run_shell("clang-tidy-cache files not in cache", f'grep -B1 "does not exist in cache" {clang_tidy_cache_log}')
```

Comment:
I'd avoid printing all the failures. It's already exported so printing in some cases thousands of lines it's not useful.

Reply:
Done in https://github.com/ClickHouse/ClickHouse/pull/83153/commits/5ad73855ceeca8a578784832b5c681fd6e13b88a